proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -259,8 +281,27 @@ message GetClusterUsageRequest {
   google.protobuf.Timestamp start_time = 1;
   // The timestamp of the ending time of a date range.
   google.protobuf.Timestamp end_time = 2;
+  // Association id is an optional owner or repository id used to filter usage.
+  oneof association_id {
```

Comment:
@cyinma I renamed this to `association_id` instead of entity from our convo. Might not be the best term so open to any suggestions here.

Reply:
are there cases that we'd want both org id _AND_ repository id?

Reply:
Given the design of the chart I don't think were would be a situation where we would want both

Reply:
renamed to `resource_id` instead